malasakit-django/cafe/settings.py

```diff
@@ -83,8 +84,10 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'pcari',
+        'USER': 'root',
+        'PASSWORD': os.environ.get('mysql_pass'),
```

Comment:
I believe the main way an attacker can read environment variables is if they have read access to `.bashrc`, by which point everything is compromised anyways. But I agree we should look into making the app more secure